documentdb/spring-data-azure-documentdb/src/test/java/com/microsoft/azure/spring/data/documentdb/repository/repository/config/DocumentDbRepositoryConfigurationExtensionUnitTest.java

```diff
@@ -29,7 +30,7 @@
     ResourceLoader loader = new PathMatchingResourcePatternResolver();
     Environment environment = new StandardEnvironment();
     RepositoryConfigurationSource configurationSource = new AnnotationRepositoryConfigurationSource(metadata,
-            EnableDocumentDbRepositories.class, loader, environment);
+            EnableDocumentDbRepositories.class, loader, environment, new SimpleBeanDefinitionRegistry());
 
```

Comment:
yes it is from 1.5.6 we need to specify BeanDefinitionRegistry as an additional parameter. Spring boot has changed the signature of this method.